src/config.ts

```diff
@@ -2,12 +2,22 @@ import * as vscode from 'vscode'
 import semver from 'semver'
 import type { WorkspaceConfiguration, WorkspaceFolder } from 'vscode'
 import type { ResolvedConfig } from 'vitest'
-import { configDefaults } from 'vitest/config'
 import { isDefinitelyVitestEnv, mayBeVitestEnv } from './pure/isVitestEnv'
 import { getVitestCommand, getVitestVersion, isNodeAvailable } from './pure/utils'
 import { log } from './log'
 export const extensionId = 'zxch3n.vitest-explorer'
 
+// Copied from https://github.com/vitest-dev/vitest/blob/main/packages/vitest/src/defaults.ts
+// "import { configDefaults } from 'vitest'" throws unexpected URL error
```

Comment:
This statement throws "Invalid URL" error during activation (from `node_modules/vite/dist/node-cjs/publicUtils.cjs`.  That file includes codes like below:

```js
const { version: version$2 } = JSON.parse(fs$1.readFileSync(new URL('../../package.json', (typeof document === 'undefined' ? require('u' + 'rl').pathToFileURL(__filename).href : (_documentCurrentScript && _documentCurrentScript.src || new URL('node-cjs/publicUtils.cjs', document.baseURI).href)))).toString());
```
It won't work in the bundled `dist/extension.js`. I couldn't find any solution and gave up. Are anyone knows something?

Reply:
I would expect Vite to not be bundled with this extension, maybe it can be externalized/tree-shaken?

Reply:
`vitest/config` doesn't actually import anything from Vite, so I'm not sure why it even gets bundled here 🤔

Reply:
It looks like tsup/esbuild's default tree-shaking is not doing much and keeps unused `require("vite")`. Actually tsup supports rollup-based tree-shaking https://tsup.egoist.dev/#tree-shaking, so this config seems to work but rebuild slows down a bit.

```ts
// tsup.config.ts
import { defineConfig } from "tsup"

export default defineConfig({
  entry: ["./src/extension.ts"],
  external: ["vscode", "vite"],
  treeshake: true,
})
```

Personally, just inlining `defaultInclude` and `defaultExclude` is equally fine as well.

Reply:
tsup also supports plugins where we can manually remove vite import

Reply:
Do you mean esbuild plugins?

Reply:
Yes

Reply:
It is ideal to import the original config from vitest. I think it's problem of vitest bundle. `new URL` with relative path resolution won't work imported bundle. Anyway, can it be acceptable?

Reply:
Ok, let's keep hardcoded config for now